### DISCUSSION TITLE
How to start dpgen with a deepmd model

### DISCUSSION THREAD

**Discussion by goodluck-xyz** [deepmodeling/dpgen#1547] [Q&A]
![image](https://github.com/deepmodeling/dpgen/assets/54764135/4c366455-93cc-459c-815e-b6166708fabf)
I try to open the `training_init_model` like this and meet a error. The error file is this. It is from iter.000000/00.train/000
[train.log](https://github.com/deepmodeling/dpgen/files/15288413/train.log)

**Comment by njzjz** [comment]
It seems the versions mismatch. In this case, you may consider restarting from the frozen model, instead of the checkpoints.
